lib/install/validate/validator.go

```diff
@@ -548,6 +548,9 @@ func (v *Validator) checkDatastoresAreWriteable(ctx context.Context, conf *confi
 	validStores := make(map[types.ManagedObjectReference]*object.Datastore)
 	// remove any found datastores from requested datastores
 	for _, cds := range clusterDatastores {
+		var ds mo.Datastore
```

Comment:
I think the requirements above are OK.
The limited customer feedback I have gathered on Volumes indicates that _"vic-machine error if no "default" volume store is provided"_ will be especially useful as many do not realize this will break anonymous volumes and images using them.